### PR TITLE
example: add a mesh on globe

### PR DESCRIPTION
**Pull request by gliegard**
Add an example that places a simple mesh on the globe.
example-positionglobe

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -80,6 +80,11 @@
             <img src='screenshots/gpx.jpg'>
             <iframe src="" style="display: none;"></iframe>
         </div>
+        <div class="exampleContainer">
+            <h3><a href='./positionGlobe.html' target='_blank'>Position a mesh</a></h3>
+            <img src='screenshots/globe.jpg'>
```
Comment (autra) [inline examples/index.html]: Can you provide a screenshots of the initial view of your example?

**Comment by autra** [comment]
Otherwise looking good :-)

File: examples/index.html (hunk 1 above)
Comment (gliegard) [reply examples/index.html]: yes, it's done.

**Review by autra — APPROVED** [review]
Thanks!

**Pull request merged**